x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/type/ExtendedJDBCType.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.sql.proto.type;
+
+import java.sql.JDBCType;
+import java.sql.SQLType;
+import java.sql.Types;
+
+/**
+ * Provides ODBC-based codes for the missing SQL data types from {@link Types}/{@link JDBCType}.
+ */
+enum ExtendedJDBCType implements SQLType {
```

Comment:
Why do we need also: https://github.com/elastic/elasticsearch/pull/35521/files#diff-479597d9c5bcd22d94ba26ef972fa181 ?